### PR TITLE
overthebox: Better speedtest

### PR DESCRIPTION
**Pull request by PouuleT**
Dirty fix edge cases when iperf is not ready to accept new connections
with a sleep
Add option to the script in order to test a specifix interface

### CODE REVIEW

File: overthebox/files/bin/otb-action-speedtest [hunk 1]
```diff
@@ -103,6 +103,7 @@ test_interface() {
 	download=$(echo "$result" | jq ".end.sum_received.bits_per_second / 1024")
 
 	printf "%s Kbps\n" "$download"
+	sleep 1
```
Comment (angt) [inline overthebox/files/bin/otb-action-speedtest]: img
